meson.build

```diff
@@ -153,8 +153,16 @@ executable(
 subdir('data')
 subdir('po')
 
-gnome.post_install(
+# gnome.post_install() is available since meson 0.59.0
+# Distributions use their own tooling (e.g. postinst, triggers, etc)
+# so it is okay if the post_install() is not run on distro builds
+m_ver = meson.version().split('.')
+if m_ver[0].to_int() > 0 or m_ver[1].to_int() > 58
```

Comment:
No!!!

Please, use `meson.version().version_compare('>=0.59.0')`. :)

Reply:
fair enough, fixed!

Reply:
Thanks. :)

Reply:
Thank you! I didn't know about that before. I agree, it is a nicer function.